### PR TITLE
Translate bash-handbook to zh-TW

### PR DESCRIPTION
**Pull request by neighborhood999**
Translations to zh-TW.

### CODE REVIEW

**Comment by denysdovhan** [comment]
Awesome! Can anybody check this translation? It'd be nice to have a small review before I merge.

**Comment by neighborhood999** [comment]
@denysdovhan I have already check this translation. :smile:

**Comment by denysdovhan** [comment]
@neighborhood999 I have no doubts, but it's not enough. There must be someone who will check this translation, because we can't find our mistakes objectively.

P.S: I'll shout on Twitter to find this _someone_ who will do this, but feel free to ask your folks 😉

**Comment by neighborhood999** [comment]
@denysdovhan OK, I know. Thank you.

**Comment by neighborhood999** [comment]
@denysdovhan already have other people helped review my translate, please check and thanks @aweimeow. :laughing:

File: translations/zh-TW/README.md [hunk 1]
```diff
@@ -0,0 +1,943 @@
...
+
+**環境變數**執行在目前 shell session 可以讓任何程式或 script 存取的變數。建立它們與區域變數類似，但使用 `export` 作為前綴
+
+```bash
+export GLOBAL_VAR="I'm a global variable"
+```
+
+bash 中有_許多_全域變數。你會經常遇到這些全域變數，以下是最常使用到的變數查閱表︰
+
+| 變數         | 描述                                                          |
```
Comment (denysdovhan) [inline translations/zh-TW/README.md]: Please align the right pipes in tables.

File: translations/zh-TW/README.md [hunk 2]
```diff
@@ -0,0 +1,943 @@
...
+| `$PS2`       | 次要的提示字串                                                |
+
+根據這個[連結](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_03_02.html#sect_03_02_04)，可以查看更多在 Bash 內的環境變數列表。
+
+## 命令列引數
+
+**命令列引數**是當程式執行時，變數根據執行程式時，程式名稱後所跟隨的參數而得。下面表格列出了當你在程式執行時，命令列引數和其他特殊變數它們所代表的意義。
+
+
+| 參數           | 描述                                                          |
```
Comment (denysdovhan) [inline translations/zh-TW/README.md]: Here the same as above

File: translations/zh-TW/README.md [hunk 3]
```diff
@@ -0,0 +1,943 @@
...
+
+最重要的是我們瞭解如何使用這個強大和複雜的工具。
+
+## Streams
+
+Bash 接收輸入並將輸出作為序列或字元的 **streams** 。這些 streams 可能會被重導向到檔案或其他 streams。
+
+這裡有三個描述：
+
+| 代碼 | 描述符     | 描述                 |
```
Comment (denysdovhan) [inline translations/zh-TW/README.md]: The same as above

File: translations/zh-TW/README.md [hunk 4]
```diff
@@ -0,0 +1,943 @@
...
+| :--: | :--------: | :------------------- |
+| `0`  | `stdin`    | 標準輸入             |
+| `1`  | `stdout`   | 標準輸出             |
+| `2`  | `stderr`   | 錯誤輸出             |
+
+重導讓我們可以控制命令的輸入來自哪裡，和輸出到哪去。在重導向 streams 時會用到以下運算子：
+
+| 運算子   | 描述                                         |
```
Comment (denysdovhan) [inline translations/zh-TW/README.md]: The same

File: translations/zh-TW/README.md [hunk 5]
```diff
@@ -0,0 +1,943 @@
...
+
+有兩種不同條件陳述：`if` 和 `case`。
+
+## Primary 和組合表達式
+
+在 `[[ ]]`（或 `sh` 中的 `[ ]`）中的表達式稱為 **測試命令** 或 **primaries**。這些表達式幫助我們表明條件的結果。在下表中，我們使用 `[ ]`，因為它也可以在 `sh` 執行。這裡有關於在 bash 中，[雙引號和單引號的差別](http://serverfault.com/a/52050)的一些回答。
+
+**檢查檔案系統中的檔案或目錄的狀態：**
+
+| Primary       | 涵義                                                         |
```
Comment (denysdovhan) [inline translations/zh-TW/README.md]: Here again

File: translations/zh-TW/README.md [hunk 6]
```diff
@@ -0,0 +1,943 @@
...
+
+shell 提供了 debug script 的工具。如果我們想要在 debug 模式執行 script，我們在 script 的 shebang 使用一個特別的選項：
+
+```bash
+#!/bin/bash options
+```
+
+這個選項是改變 shell 行為的設定。下面是一些可能對你有幫助的選項清單：
+
+| 簡寫  | 名稱        | 描述                                                      |
```
Comment (denysdovhan) [inline translations/zh-TW/README.md]: And again :)

**Comment by denysdovhan** [comment]
LGTM, but please, fix tables.

**Comment by neighborhood999** [comment]
@denysdovhan hey sorry, I don't understand your means.



Like this？This my original README

**Comment by aweimeow** [comment]
@neighborhood999 I'd fixed it, and later will sent PR to you again

**Comment by denysdovhan** [comment]
@neighborhood999 yes, make them pair to pair.

**Comment by neighborhood999** [comment]
@denysdovhan I have already align the right pipes in tables as above picture and here：


Seems chinese character problem, I have already strive align table.

**Comment by neighborhood999** [comment]
@denysdovhan I have already align the right pipes in tables as above picture and here：


Seems chinese character problem, I have already strive align table.

**Comment by denysdovhan** [comment]
Well, okay, it's doesn't matter. I'm gonna merge. Please, subscribe to the repo to be updated. I suppose you will update your translation if we change something in original text.

**Comment by denysdovhan** [comment]
[Published as `v2.7.0`](https://twitter.com/denysdovhan/status/735508759188013056)! 
@neighborhood999 @aweimeow, thank you all!

**Pull request merged**